addons/medical_damage/functions/fnc_woundsHandlerSQF.sqf

```diff
@@ -93,17 +93,22 @@ private _bodyPartVisParams = [_unit, false, false, false, false]; // params arra
             _bodyPartVisParams set [[1,2,3,3,4,4] select _bodyPartNToAdd, true]; // Mark the body part index needs updating
 
 
-            // The higher the nastiness likelihood the higher the change to get a painful and bloody wound
-            private _nastinessLikelihood = linearConversion [0, 20, (_woundDamage / _thresholdWoundCount), 0.5, 30, true];
-            private _bleedingModifier = 0.25 + 8 * exp ((random [-4.5, -5, -6]) / _nastinessLikelihood);
-            private _painModifier = 0.05 + 2 * exp (-2 / _nastinessLikelihood);
+            // Config specifies bleeding and pain for worst possible wound
+            // Worse wound correlates to higher damage, damage is not capped at 1
+            // Damage to limbs is scaled higher than body/torso by engine
+            // Anything above this is guaranteed worst wound possible
+            private _worstDamage = [2, 4] select (_bodyPartNToAdd > 1);
 
-            private _bleeding = _injuryBleedingRate * _bleedingModifier;
+            // More wounds means more likely to get nasty wound
+            private _bleedModifier = linearConversion [0.1, _worstDamage, _woundDamage * _i, 0.25, 1, true];
```

Comment:
Something we may want to change here is
```suggestion
            private _bleedModifier = linearConversion [0.1, _worstDamage, _woundDamage * (1 + random (_i - 1)), 0.25, 1, true];
```

which just makes it so that beyond 1 wound there's an increased *chance* for more nasty wounds, rather than steadily increasing nastiness. This would mean if the wound damage is 0.66 there a chance you can get 3 medium wounds best case, or 1 medium wound and 2 large wounds worst case. Whereas currently it would result in guaranteed 2 medium wounds and 1 large wound.

Reply:
Something else to consider is we could remove the upper limit on the linear conversion. This would mean wound damage values greater than the `worstDamage` can actually cause wounds which bleed more than the configs.